File: Beecrowd/1387 Og/main.py
L, R = 1, 1

while L != 0 or R != 0:
  entrada = [int(i) for i in input().split()]
  L = entrada[0]
  R = entrada[1]
  if L != 0 or R != 0:
    soma = L + R
    print(soma)

File: Beecrowd/1387 Og/main.c
#include <stdio.h>

int main()
{
    int L, R, soma;

    do
    {
        scanf("%d %d", &L, &R);

        if (L != 0 || R != 0)
        {
            soma = L + R;

            printf("%d\n", soma);
        }

    } while (L != 0 || R != 0);

    return 0;
}